### DISCUSSION TITLE
2024-03-06 DDEV Advisory Group: 2023 review, 2024 planning

### DISCUSSION THREAD

**Discussion by rfay** [ddev/ddev#5757] [DDEV Advisory Group]
Previous meeting: 
* #5483

Next meeting: 1 May 2024

## Agenda

* Turn on recording, ask for a minutes-taker

### Review of DDEV 2023

* [2023 Review Blog Post](https://ddev.com/blog/2023-review/)
* Financials

  * [DDEV Foundation Balance Sheet 2023.pdf](https://github.com/ddev/ddev/files/14483637/DDEV.Foundation.Balance.Sheet.2024-03-04-15_25.pdf)
  * [DDEV Foundation Profit and Loss 2023.pdf](https://github.com/ddev/ddev/files/14483638/DDEV.Foundation.Profit.and.Loss.2024-03-04-15_24.pdf)
  * Bank balance as of 2023-03-04: USD$16,515.

### DDEV 2024 plans, goals, discussion

[2024 Plans Blog Post](https://ddev.com/blog/2024-plans)

### Financial Matters: 
  * Mike Anello (@ultimike) is now a signer on the DDEV Foundation bank account. This means that if Randy were not available in any way, Mike could do the right thing with the funds in the account. This is a small first step toward more formal governance with DDEV.

* Anything on anybody's mind!

## Meeting Notes

https://docs.google.com/document/d/1c4j-mnjUhGbVk07miNbvEeOmyHsDPIHDXvhYjsWmawQ/edit?usp=sharing

## Recording

https://youtu.be/Qq4G0j_8wFY
[Chat.txt](https://github.com/ddev/ddev/files/14515198/meeting_saved_chat.txt)

## Randy's Patagonia Bikepacking

If you're interested, [Randy's Chilean Patagonia Bikepacking Slideshow](https://studio.youtube.com/video/5zPWGRfNW-k/edit)

**Comment by chrisfromredfin** [comment]
I don't think I can update the main thread, but here's a link to the meeting notes. Now that the meeting is over, I've adjusted permissions on the document to "Anyone with the link: Can comment" (whereas during the meeting it was "edit" and was shared in the Zoom chat).

https://docs.google.com/document/d/1c4j-mnjUhGbVk07miNbvEeOmyHsDPIHDXvhYjsWmawQ/edit

**Reply by rfay** [reply]
Thanks, updated.
